com.epam.zoo/src/Food.java

```diff
@@ -0,0 +1,3 @@
+public enum Food {
+    fruits, grass, meat
```

Comment:
Rename to FoodType and use in Animal class
Correct format for your enums:
FRUIT("Fruit"), GRASS("Grass"), MEAT("Meat")